concent_api/utils/logging.py

```diff
@@ -81,23 +84,27 @@ def log_timeout(message: Message, client_public_key: bytes, deadline: int):
 
 
 @replace_element_to_unavailable_instead_of_none
-def log_empty_queue(endpoint: str, client_public_key: bytes):
+def log_empty_queue(
+    logger,
+    endpoint: str,
+    client_public_key: bytes
+):
     logger.info('A message queue is empty in `{}()` -- CLIENT PUBLIC KEY: {}'.format(
         endpoint,
         client_public_key,
     ))
 
 
 @replace_element_to_unavailable_instead_of_none
-def log_400_error(endpoint: str, client_public_key: bytes, message: Message):
-    if message is not None:
-        task_id      = get_task_id_for_logging(message)
-        subtask_id   = get_subtask_id_for_logging(message)
-        message_type = type(message).__name__
-    else:
-        task_id      = 'not available'
-        subtask_id   = 'not available'
-        message_type = 'not available'
+def log_400_error(
+    logger,
+    endpoint: str,
+    client_public_key: bytes,
+    message: Message
+):
+    task_id = _get_field_value_from_messages_for_logging(MessageFields.task_id, message)
+    subtask_id = _get_field_value_from_messages_for_logging(MessageFields.subtask_id, message)
+    message_type = type(message).__name__ if message is not None else 'UNAVAILABLE'
```

Comment:
i think it should be part of `replace_element_to_unavailable_instead_of_none` decorator

Reply:
done